### ISSUE TITLE
Add support for Preserve to boost timers

### ISSUE THREAD

**Issue by SomeoneWithAnInternetConnection** [runelite/runelite#1059]
We'd need to nail down what the threshold for the effect kicking in is. [This reddit thread](https://www.reddit.com/r/2007scape/comments/5p4lvi/can_someone_explain_how_stat_boosting_works_now/dcod4yf/?st=jf2z23mt&sh=d5357e53) has some accurate-looking information about the stat-drain stuff in general:
```
before, every 1 minute your stats drop. logging out refreshes this timer

now every 12 seconds your stats get marked and after 5 marks,
or 6 with preserve, your stats drop. logging out resets the 12 second timer
but not the marks, so if you log out every 12 seconds your stats will never drop,
but if you wait 50 seconds, after which youll have 4 marks,
when you relog in youll still have those 4 marks so 12 seconds later your stats will drop
```

Discussion in discord has made it sound like it's something like "if you've had preserve on for the last 15 seconds, and a normal stat boost drain would have happened this tick, delay that stat boost drain by 30 seconds". Given the 12 second threshold for preserving a boost by logging out, the preserve threshold *may* be 12 seconds, rather than 15.

**Comment by ShaunDreclin** [comment]
How does the current boosts plugin work? If it doesn't take the reworked system with "marking" each boost every 12 seconds into account, it'd be nice to get this even without the preserve prayer.

**Comment by deathbeam** [comment]
Fixed in #3530
